app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
```

Comment:
I think you'll probably need to define a post somewhere here to actually connect comments and posts?